tests/vip-support/test-user-proxied.php

```diff
@@ -32,23 +45,37 @@ public function test__superadmin_filter(): void {
 			'display_name' => 'Regular User',
 		] );
 
-		$this->assertFalse( is_super_admin( $regular_user_id ) );
```

Comment:
Interesting code paths won't be invoked from `is_super_admin()` unless we call it from multisite, which sits behind a constant with no option to mock them. Or I just don't know how to do that.

Reply:
Great catch!